### DISCUSSION TITLE
Add the option to send notes to a webhook (for example make.com)

### DISCUSSION THREAD

**Discussion by blocoder** [alangrainger/share-note#121] [Vote for upcoming features]
It would be great if I could send certain notes to a webhook. Because then I could — for example with make.com — process the note texts automatically.

Thank you very much!
Peter

**Comment by alangrainger** [comment]
Hi @blocoder - appreciate the suggestion, but that's out of scope for this plugin. 

However that would be very easy to do just with Templater or CustomJS:

1. Read the note content with `app.vault.getAbstractFileByPath(path); app.vault.read(file)`
2. Send to your webhook with `requestUrl()`

**Reply by blocoder** [reply]
Thanks for your prompt response.
I will try out your suggestion. Many thanks for that too, @alangrainger!
